src/views/DownloadPackedOrders.vue

```diff
@@ -54,17 +54,17 @@
 
           <ion-item :key="field" v-for="(value, field) in fieldMapping" v-show="!selectedData[field]">
             <ion-button slot="start" fill="clear" @click="updateSelectedData(field)">
-              <ion-icon color="success" :icon="addCircleOutline"/>
+              <ion-icon color="success" :icon="addCircleOutline" />
```

Comment:
Revert all this unwanted changes, like removing a space or adding a new space etc